dbms/src/Flash/Mpp/MPPTask.h

```diff
@@ -147,12 +158,11 @@ class MPPTask : public std::enable_shared_from_this<MPPTask>
 
     MPPTaskScheduleEntry schedule_entry;
 
+    ProcessListEntryHolder process_list_entry_holder;
```

Comment:
Does process_list_entry_holder need to be moved in front of `context`? Because `Context` will hold ProcessListEntry

Reply:
ok